quipucords/scanner/network/tests_network_connect.py

```diff
@@ -124,11 +124,12 @@ def setUp(self):
 
         self.source = Source(
             name='source1',
+            hosts='["1.2.3.4", "1.2.3.5"]',
```

Comment:
We probably should set up an additional source that does not have `exclude_hosts` to ensure that the we have tested the case where it is not specified. IE. We need to make sure that everything has been tested with & without `exclude_hosts`.

For example when we call the `get_exclude_hosts` from the source model, we need to make sure that it will return [] if the source was created without `exclude_hosts`.